moveit_server:
  ros__parameters:
    collision_object_frame: "world"
    base_frame_id: "link_base"
    end_effector_frame_id: "link_eef"
    orientation_tolerance:
      x: 0.75
      y: 0.75
      z: 0.75
    planning_attempts: 5
    planning_time: 2.0
    goal_tolerance: 0.005
    goal_orientation_tolerance: 0.005
    velocity_scaling_factor: 0.2
    acceleration_scaling_factor: 0.4
